templates/shared_services/gitea/template_schema.json

```diff
@@ -26,6 +26,18 @@
       "description": "Long form description of the workspace service, in markdown syntax",
       "default": "The Gitea shared service is used for mirroring external Git repositories. For instructions on how to create Git mirrors see [https://docs.gitea.com/usage/repo-mirror](https://docs.gitea.com/usage/repo-mirror).",
       "updateable": true
+    },
+    "is_exposed_externally": {
+      "$id": "#/properties/is_exposed_externally",
+      "type": "boolean",
+      "title": "Expose externally",
+      "description": "Is the Gitea accessible from outside of the TRE network.",
+      "default": false
```

Comment:
This default is different to the model's default - intentional?

Reply:
Yes, as its not configured in the terraform to enable it to be exposed externally.